src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/Example5.java

```diff
@@ -0,0 +1,41 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="CommentsIndentation"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+// xdoc section -- start
+public class Example5 {
+  void testMethod(String a) {
+    switch(a) {
+      case "1":
+        int j = 7;
+        // it is OK
+        break;
+      case "2":
+        int k = 7;
+        break;
+      // it is OK
+      case "3":
+        if (true) {}
+        // violation below, 'Comment has incorrect indentation level 12'
+            //It is not OK
+        break;
+      case "4":
+        break;
```

Comment:
Please remove extra cases that don't show anything about behavior of this Check

Reply:
Done

Reply:
I still see it

Reply:
I still see it

Reply:
Done